circle.yml

```diff
@@ -236,6 +236,8 @@ commands:
           module_target: "libnavui-maneuver"
       - store-results:
           module_target: "libnavui-tripprogress"
+      - store-results:
+          module_target: "libnavui-voice"
```

Comment:
why do we want this to be another module? in other words, what exactly makes voice a good candidate for modularity?

is this to help compile time, will customers create their own voice modules, does the sdk support "bring your own voice instructions", or something else?